Source/FullScreenPokemon.ts

```diff
@@ -6667,6 +6667,81 @@ module FullScreenPokemon {
                     }));
         }
 
+        /**
+         * Cutscene for a Scratch attack in battle.
+         * 
+         * @param FSP
+         * @param settings   Settings used for the cutscene.
+         * @param args   Settings for the routine.
+         */
+        cutsceneBattleAttackScratch(FSP: FullScreenPokemon, settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+            let defenderName: string = args.defenderName;
+            let defender: IThing = <IThing>FSP.BattleMover.getThing(defenderName);
+            let dt: number = 1;
+            let direction: number = defenderName === "opponent" ? -1 : 1;
+            let scratchLine: string = "ScratchLine";
+            let startX: number;
+            let startY: number;
```

Comment:
Can you put `startX` and `startY` below the other `lets`? They only get assigned after everything else, so it makes sense to keep them closer to their logic.